priv/repo/migrations/20180824204501_create_uploads_table.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Ret.Repo.Migrations.CreateUploadsTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:uploads, prefix: "ret0", primary_key: false) do
+      add :upload_id, :bigint, default: fragment("ret0.next_id()"), primary_key: true
+      add :upload_uuid, :string
+      add :uploader_account_id, :integer, null: false
+      # TODO BP: These should probably be non-null
+      add :state, :string
+      add :size, :bigint
```

Comment:
probably should add the mime type here

Reply:
ie, for persistent uploads, instead of having a `meta` file, we'll rely upon the database record. we should talk through if we should just convert everything to using the database though, probably would be better.